utilities/transactions/write_prepared_txn_db.cc

```diff
@@ -168,6 +168,10 @@ void WritePreparedTxnDB::Init(const TransactionDBOptions& /* unused */) {
       new std::atomic<CommitEntry64b>[COMMIT_CACHE_SIZE] {});
 }
 
+#define ROCKSDB_LOG_DETAILS(LGR, FMT, ...) \
+  ;  // due to overhead by default skip such lines
+// ROCKS_LOG_DEBUG(LGR, FMT, ##__VA_ARGS__)
```

Comment:
Initially I used ROCKS_LOG_DEBUG but it turns out that it increases the runtime by >5x and it would be much more with asan tests. I could not disable that with DNDEBUG since DEBUG is actually enabled with tests. I would have to invent yet another compile switch for this. I figured since we only need it when we are doing in-depth debugging, it is simpler to enable/disable them in the code.